test/sys/test_prctl.rs

```diff
@@ -99,6 +99,9 @@ mod test_prctl {
         prctl::set_timerslack(original).unwrap();
     }
 
+    // Loongarch need to use a newer QEMU that disabled these PRCTL subcodes/methods.
+    // So we should ignore them when testing in QEMU environments.
+    #[cfg_attr(all(qemu, target_arch = "loongarch64"), ignore)]
```

Comment:
> Loongarch64 need a newer QEMU after 7.1 version. But before QEMU supported la64, qemu has a commit that has [disabled many PRCTL subcodes](https://github.com/qemu/qemu/commit/220717a6f46a99031a5b1af964bbf4dec1310440) 3 yrs ago. This commit will cause test_prctl failed in some cases, due to these subcodes has been disabled in newer QEMU.

So due to the use of that custom cross image, the Loongarch64 CI is using a QEMU that is different from other CI, this is the reason why this `ignore` is only needed for Loongarch64, right?

Let link the QEMU commit here:

```suggestion
    // Loongarch need to use a newer QEMU that disabled these PRCTL subcodes/methods:
    // https://github.com/qemu/qemu/commit/220717a6f46a99031a5b1af964bbf4dec1310440
    // So we should ignore them when testing in QEMU environments.
    #[cfg_attr(all(qemu, target_arch = "loongarch64"), ignore)]
```

Reply:
yes. Loongarch64 needs QEMU 7.1 version, newer than the others. And I've added the link in comments.